app/views/shared/_stats.html.erb

```diff
@@ -1,59 +1,65 @@
-<div class="crayons-tabs crayons-tabs--analytics">
+<div class="crayons-tabs crayons-tabs--analytics my-8">
   <button class="crayons-tabs__item crayons-tabs__item--current" id="week-button">Week</button>
   <button class="crayons-tabs__item" id="month-button">Month</button>
   <button class="crayons-tabs__item" id="infinity-button">Infinity</button>
 </div>
 
 <div class="summary-stats">
-  <div class="card" id="readers-card"></div>
-  <div class="card" id="reactions-card"></div>
-  <div class="card" id="comments-card"></div>
-  <div class="card" id="followers-card"></div>
+  <div class="summary-stats__col crayons-card mx-5">
+    <div id="readers-card" class="py-3"></div>
```

Comment:
TIL about the upside of using `id` selectors for implementations like these ones (throughout the file):

Small refactors are easy because the JS doesn't need to be edited after updating the layout 😄